types/xmldom-reborn/index.d.ts

```diff
@@ -0,0 +1,50 @@
+// Type definitions for xmldom-reborn 1.0.5
+// Project: https://github.com/MrDatastorage/xmldom-reborn
+// Definitions by: Qubo <https://github.com/tkqubo>
+//                 Andreas Neeven <https://github.com/aneeven>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+
+declare namespace xmldom {
```

Comment:
Is there any particular reason you are defining this in a namespace and using `export =`? You could just remove the namespace and export everything at the top level.

Reply:
No particular reason, but I hope it's fine now. We're in a bit of a hurry over here, we need those types to swap an old package due to licensing issues.